aptos-move/aptos-release-builder/src/components/mod.rs

```diff
@@ -277,6 +277,44 @@ impl ReleaseConfig {
     pub fn parse(serialized: &str) -> Result<Self> {
         serde_yaml::from_str(serialized).map_err(|e| anyhow!("Failed to parse the config: {:?}", e))
     }
+
+    pub fn validate_upgrade(&self, endpoint: Url) -> Result<()> {
+        let client = Client::new(endpoint);
+        if let Some(features) = &self.feature_flags {
+            let on_chain_features = block_on(async {
+                client
+                    .get_account_resource_bcs::<aptos_types::on_chain_config::Features>(
```

Comment:
nit: Why use the fully qualified path aptos_types::on_chain_config::Features here and not just Features?

Reply:
They are different types: Features in `aptos_types` is the serialized format on chain and the one previously is more of a human readable format.